gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
-#Wed Jun 01 21:24:45 CEST 2016
+#Tue May 16 01:37:52 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionSha256Sum=0f665ec6a5a67865faf7ba0d825afb19c26705ea0597cec80dd191b0f2cbb664
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-bin.zip
+distributionSha256Sum=d84bf6b6113da081d0082bcb63bd8547824c6967fe68704d1e3a6fde822b7212
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
We have a Gradle Plugin in the project, and it is a PITA to always change it to all if you want the sources. It's 71mb vs 92mb, so ~30% increase, I thought it's ok since it is downloaded once for a user.